src/IF.Lastfm.Core/Api/TagApi.cs

```diff
@@ -44,6 +44,20 @@ public Task<LastResponse<LastTag>> GetInfoAsync(string tagName)
         }
 
         /// <summary>
+        /// Get the top albums tagged by this tag, ordered by tag count.
+        /// </summary>
+        public Task<PageResponse<LastAlbum>> GetTopAlbumsAsync(string tagName, int page=1, int itemsPerPage= LastFm.DefaultPageLength)
+        {
+            var command = new GetTopAlbumsCommand(Auth, tagName)
+            {
+                Page = page,
+                Count = itemsPerPage
+            };
```

Comment:
Also need to set the HttpClient property of the command, ```HttpClient = HttpClient```. It's a hangover from #41, not sure how's best to fix it.